packages/v4/patternfly-docs/content/accessibility/avatar/avatar.md

```diff
@@ -3,21 +3,54 @@ id: Avatar
 
-**Keyboard users** shouldn’t be able to interact with the avatar on its own. Thus, the user should not be able to focus on the avatar using **Tab** to move forward and **Tab + Shift** to move backwards through interactive elements.
+## Accessibility
 
-**Screen reader users** should be able to navigate to the avatar and it should have alternative text, since it’s an image element. You can use the `alt` prop to provide alternative text.
+To implement an accessible PatternFly **avatar**:
 
-If you’re combining an avatar with another component, make sure to check accessibility guidelines for that component as well.
+- Use the `alt` React prop to provide alternative text for the avatar image.
+- If you are using an SVG element for the avatar, [learn how to create accessible SVGs](https://www.deque.com/blog/creating-accessible-svgs/) and reference the different SVG patterns. Screen reader accessibility for SVGs varies based on the pattern being used.
+- If you’re combining an avatar with another component, make sure to check accessibility guidelines for that component as well.
 
+## Testing
 
-## To make avatar accessible:
-- You may use the `alt` React prop to provide alternative text for the avatar image.
-- If you are using an SVG element for the avatar, [learn how to create accessible SVGs](https://www.deque.com/blog/creating-accessible-svgs/) and reference the different SVG patterns. Screen reader accessibility for SVGs varies based on the pattern being used.
+At a minimum, tabs should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="avatar-a11y-checkbox-4" label="Screen reader users can navigate to the avatar." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="avatar-a11y-checkbox-1" label="The avatar image has alternative text." description={<span>Use the <code class="ws-code">alt</code> prop to provide alternative text.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="avatar-a11y-checkbox-2" label="Any SVGs used are accessible." description={<span><a href="https://www.deque.com/blog/creating-accessible-svgs/">How to create accessible SVGs</a></span>}/>
+  </ListItem>
+  <ListItem>
+    <Checkbox id="avatar-a11y-checkbox-3" label={<span>Keyboard users should not be able to interact with the avatar on its own (via <kbd>Tab</kbd> and <kbd>Tab + Shift</kbd>).</span>} />
+  </ListItem>
+</List>
+
+## React customization
 
 The following props/attributes have been added for you or are customizable in PatternFly:
 
-| React prop | React component that it should be applied to | Which HTML element it appears on in markup | Reason used |
-| -- | -- | -- | -- |
-| `alt` | Avatar | .pf-c-avatar | Provides an accessible description of the avatar as it uses an image instead of text. |
+| Prop | Applied to | Which HTML element it appears on in markup | Reason  |
+| -- | -- | -- |
+| `alt` | `Avatar` |  Provides an accessible description of the avatar. |
```

Comment:
Could we align the "reason" in both the React and HTML/CSS tables?

Also, do you think it'd be worth adding on a little more to the purpose of providing alt text? Mainly thinking of how it will be rendered if the image is unable to load properly (slow wifi/internet and such).

Reply:
Honestly, I don't think we need to go into the purpose of alt text. I can if you want, but it seems basic enough and I don't think we need to make an argument for it.

Reply:
That's a fair point! I don't have a strong opinion either way, so I'm good with not going too into it like you said